src/tasks.py

```diff
@@ -63,30 +164,47 @@ def command(
     """
     input_files = get_input_files(pipe_result, input_files or [])
     output_files = []
-    base_command = ["<REPLACE_WITH_COMMAND>"]
-    base_command_string = " ".join(base_command)
+    file_reports = []
 
     for input_file in input_files:
+        base_command = ["bulk_extractor"]
         output_file = create_output_file(
             output_path,
-            display_name=input_file.get("display_name"),
-            extension="<REPLACE_WITH_FILE_EXTENSION>",
-            data_type="<[OPTIONAL]_REPLACE_WITH_DATA_TYPE>",
+            display_name="Report_{}".format(input_file.get("display_name")),
+            extension="html",
+            data_type="html",
```

Comment:
We can skip these two, use the default data_type (we will change this in the near future) and we set the extension on the display_name above
```suggestion
```

Reply:
Done